js/components/dashboard/answer.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react'
+import { Map } from 'immutable'
+import MultipleChoiceAnswer from './multiple-choice-answer'
+
+import css from '../../../css/dashboard/answer.less'
+
+export const NoAnswer = () => null
+
+export const GenericAnswer = () => (
+  <div style={{textAlign: 'center'}}>
```

Comment:
It's still centered in the collapsed view. I disabled vertical centering in expanded view, as this icon was the only thing that was vertically centered. Student name and progress bars were at the top of the cell, I also added expanded multiple choice answer at the top. I was thinking about centering everything in expanded view (student name, expanded answers, progress bars and this icon). But since everything has different height, I'm not sure if it would look good. Also, Ethan's mock had top alignment, so I just followed that to avoid decision making. :wink:

And it would be actually pretty tricky to make scrolling container that is vertically centered when there's no overflow and overflow looks correct. I had this problem before. It centers scrollable content too, so you can't see the top part. Perhaps there's some workaround tho.